### PR TITLE
Revkn-210-Provide documentation on finding a particular upload result

### PR DESCRIPTION
**Pull request by Baffoe-Caleb**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -206,10 +206,33 @@ This endpoint loads customers by csv files
...
 
+### Responses
+
+| Code | Description                                                                                                             |
+|------|----------------------------------------------------------------------------------|
+| 200  | A report object showing which of the customer records were successfully processed| 
+| 403  | Operation not permitted for this business                                        |
+| 404  | Invalid businessId or importId supplied                                          |
+
+
```
Comment (zematrix) [inline source/index.html.md:278]: Where is the example json output? Run the `enterprise-openapi`. The example format is there now.

File: source/index.html.md [hunk 2]
```diff
@@ -183,6 +183,29 @@ This endpoint finds  customer record by its id
...
+      "phoneNo": "0247736845",
+      "phoneNoValid": true
+    }
+  ],
+  "id": "7359ba7e-a6c4-11ec-b909-0242ac120002"
+}
+```
+
+To format the CSV, the CSV file must have the following records, and must have a heading matching the following.
+First name;Last name;Phone number;Customer number
```
Comment (zematrix) [inline source/index.html.md]: This sentence should be done the exact way i sent it to you.

```
The CSV file must have the following records, and must have a heading matching the following.
```
Followed by the following in caps on a separate line.

**First name;Last name;Phone number;Customer number**

Comment (zematrix) [reply source/index.html.md]: Beneath it, add this

* `Customer number` is not required and can be empty.

File: source/index.html.md [hunk 3]
```diff
@@ -203,7 +203,7 @@ This endpoint loads customers by csv files
 
 The CSV file must have the following records, and must have a heading matching the following.
 
-**First name;Last name;Phone number;Customer number**
+**FIRST NAME;LAST NAME;PHONE NUMBER;CUSTOMER NUMBER**
```
Comment (zematrix) [inline source/index.html.md]: I didn't ask you to capitalize it. Its `First name`, not `FIRST NAME`. Pay attention. It should be as i sent you.

Comment (Baffoe-Caleb) [reply source/index.html.md]: Please this is what you said in the comment "Followed by the following in caps on a separate line." I was even confused

File: source/index.html.md [hunk 4]
```diff
@@ -183,6 +183,35 @@ This endpoint finds  customer record by its id
 
 This endpoint loads customers by csv files
 
+>**Example body for uploading a csv file containing customer records**
+
```
Comment (zematrix) [inline source/index.html.md:187]: What you have here is not the body. It's the response to the operation. The body is the CSV file. So label it appropriately

**Review by zematrix — APPROVED** [review]

**Pull request merged**